migration/data/seedTokens.ts

```diff
@@ -1247,6 +1249,15 @@ const seedTokens: ITokenData[] = [
     networkId: NETWORK_IDS.MORDOR_ETC_TESTNET,
   },
   // TODO: Add solana token
+  {
+    name: 'Solana native token',
+    symbol: 'SOL',
+    address: '0x0000000000000000000000000000000000000000',
+    decimals: 18,
```

Comment:
that one SOL can be divided into 1,000,000,000 (one billion) smaller units. These smallest units are often referred to as lamports
```suggestion
    decimals: 9,
```